ftests/parallel/test_remote.py

```diff
@@ -11,7 +11,7 @@
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
+#
```

Comment:
I suggest to revert the change of white space.  But it's not critical so I am fine if you want to remove the space in this change.

In general a style change like this may cause problem when merging other branches.  For now it's fine because we don't have many branches in parallel, and this is a very straightforward patch.